.eslintrc.js

```diff
@@ -1,16 +1,11 @@
 module.exports = {
   root: true,
   parser: '@typescript-eslint/parser',
-  plugins: ['@typescript-eslint'],
   extends: [
-    'plugin:@typescript-eslint/recommended',
     'kentcdodds',
-    'kentcdodds/jest',
```

Comment:
This was the origin of the issue. I'm not sure exactly why, but since we don't have jest tests yet, I'm removing this one.

Then the other extends/plugins were causing a conflict since `kentcdodds` already set them up. I've removed duplicated ones and everything seems fine now.